foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -1134,7 +1550,43 @@ MobileCore.resetIdentities();
 ```
 {% endtab %}
 
-{% tab title="iOS \(ACP 2.x\)" %}
+{% tab title="iOS (AEP 3.x)" %}
+**Swift**
+
+### resetIdentities
```

Comment:
Missing description.

Reply:
The description is found above like all the other APIs. "The `resetIdentities` method requests that each extension resets the identities it owns. Each extension responds to this request uniquely.". Is that what you mean?

Reply:
That sounds good, the top level description is sufficient. Once we move the API name `resetIdentities` at top level, we can remove the name inside the tabs, and keep only syntax & example (see https://aep-sdks.gitbook.io/docs/foundation-extensions/places/places-usage-reference as an example for the format).